### PR TITLE
Abricate version 0.5

### PR DESCRIPTION
**Pull request by Slugger70**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This will create a recipe for the recently released abricate v0.5

### CODE REVIEW

**Comment by bgruening** [comment]
@druvus can you have a look on this please?

**Comment by bgruening** [comment]
@Slugger70: @druvus has still a dependency on perl and a few more tests included, maybe you can add this as well?

File: recipes/abricate/meta.yaml [hunk 1]
```diff
@@ -17,14 +17,16 @@ requirements:
   - zlib
   run:
   - libgcc # [linux]
-  - perl-threaded
```
Comment (druvus) [inline recipes/abricate/meta.yaml]: You should probably keep perl as a dependency

**Comment by druvus** [comment]
I added the extra tests to catch missed dependencies needed by helper scripts. Would probably be good to have them but not necessary.

**Comment by druvus** [comment]
The main issue I saw with the 0.3 version of the recipe is the unspecific copying into $PREFIX. This way it is cluttered by unnecessary files (README.md & LICENSE) and two new folders (db & scripts). The script assumes by default (if not set in the prompt) that the database is located `../db` in relation to the script. 

@jerowe @bgruening do you have some recommendation how we could package this in a better way?

**Comment by bgruening** [comment]
ping @Slugger70

**Comment by tseemann** [comment]
@druvus 
if you want to change default `--db` it will need to be patched in `abricate` and `abricate-get_db`.  These are the only 2 scripts now in version 5.0

**Comment by tseemann** [comment]
@Slugger70 maybe add `abricate --check` next time to the test suite.

**Comment by bgruening** [comment]
Let's add it now and either increase the build number to trigger a test or just add it for next time.

**Comment by tseemann** [comment]
Abricate 0.6 was released  - https://github.com/tseemann/abricate/releases

*  need --setupdb before it will work

@Slugger70

**Pull request merged**